### ISSUE TITLE
By default, ignores some undefined references like: onsole.lg()

### ISSUE THREAD

**Issue by groovy9** [typescript-language-server/typescript-language-server#438]
I'm guessing this is a configuration issue, but with the default install of typescript-language-server and the default install of the Helix editor, the editor's language server integration works as advertised except for one glaring problem.  I've looked through the options in the config object but nothing sounds applicable here.

It catches this:

`
// says "Unexpected keyword or identifier"
`
`
unction foo() {
}
`

but does not catch:
`
// no error dot, no message
`
`
onsole.lg('foo')
`
`
anyRandomThing
`

How do I get it to catch these seemingly obvious errors?

**Comment by rchl** [comment]
Create a `tsconfig.json` file in the root of your project. Some example settings could include:

```js
{
    "compilerOptions": {
        "baseUrl": ".",
        "target": "ES2020",
        "module": "ES2020",
        "esModuleInterop": true,
        "moduleResolution": "node",
        "noEmit": true,
        "checkJs": true,
        "allowSyntheticDefaultImports": true,
        "strict": true,
        "resolveJsonModule": true,
    },
}
```

**Comment by groovy9** [comment]
I'd already created the tsconfig.json file with '`tsc --init` and had assumed the commented-out options were just documenting the  internal defaults.  Nope!  I had to uncomment the checkJs option and now it's working.  Thanks a bunch.
